src/webview/MessageList.js

```diff
@@ -78,7 +82,12 @@ export type BackgroundData = $ReadOnly<{
 }>;
 
 type SelectorProps = {|
+  // Data about the user, the realm, and all known messages.
```

Comment:
This seems likely to diverge confusingly from its source.

Reply:
Edited to be in terms of the other side of the distinction. (Though in practice I think this description is likely to be pretty stable too.)